src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/DefaultGATKVariantAnnotationArgumentCollection.java

```diff
@@ -62,6 +62,19 @@ public DefaultGATKVariantAnnotationArgumentCollection(final List<String> default
     @Argument(fullName = StandardArgumentDefinitions.DISABLE_TOOL_DEFAULT_ANNOTATIONS, shortName = StandardArgumentDefinitions.DISABLE_TOOL_DEFAULT_ANNOTATIONS, doc = "Disable all tool default annotations", optional = true)
     public boolean disableToolDefaultAnnotations = false;
 
+    /**
+     * You can use the -AX argument in combination with this one to exclude specific annotations. Note that some
+     * annotations may not be actually applied if they are not applicable to the data provided or if they are
+     * unavailable to the tool (e.g. there are several annotations that are currently not hooked up to
+     * HaplotypeCaller). At present no error or warning message will be provided, the annotation will simply be
+     * skipped silently. You can check the output VCF header to see which annotations were actually applied (although
+     * this does not guarantee that the annotation was applied to all records in the VCF, since some annotations have
+     * additional requirements, e.g. minimum number of samples or heterozygous sites only -- see the documentation
+     * for individual annotations' requirements).
+     */
+    @Argument(fullName="use-all-annotations", doc="Use all possible annotations (not for the faint of heart)", optional=true)
+    protected Boolean USE_ALL_ANNOTATIONS = false;
```

Comment:
I'm unsure what to do about that one, as this is just a holdover from the current version of VariantAnnotator which took the argument from gatk3, it seems we would be breaking the current version of the tool if we changed it so.

Reply:
@jamesemery The comment was about the instance *variable* name `USE_ALL_ANNOTATIONS`, not the command line argument name. When it was in VariantAnnotator it also had the shortName "all" (correctly removed now since I think we generally don't use them). So I don't think it will be a breaking change.